templates/severity_inconsistency.html

```diff
@@ -0,0 +1,27 @@
+<p>The following {{ plural('bug has', data, pword='bugs have') }} inconsistency between the bug severity and Whiteboard severity flag. {{ plural('Owner', data) }} got {{ plural('a needinfo request', data, pword='needinfo requests') }} to bump the severity:
```

Comment:
The plural word is missing for "Owner"

Reply:
Without specifying the plural word, the `plural` function will add an *s* at the end of the singular word.